inlong-sort/sort-end-to-end-tests/src/test/resources/flinkSql/clickhouse_test.sql

```diff
@@ -0,0 +1,33 @@
+CREATE TABLE test_input1 (
+    `id` INT primary key,
+    name STRING,
+    description STRING
+) WITH (
+    'connector' = 'mysql-cdc-inlong',
+    'hostname' = 'mysql',
+    'port' = '3306',
+    'username' = 'inlong',
+    'password' = 'inlong',
+    'database-name' = 'test',
+    'table-name' = 'test_input1',
+    'scan.incremental.snapshot.chunk.size' = '4',
+    'scan.incremental.snapshot.enabled' = 'false'
+);
+
+CREATE TABLE test_output1 (
+    `id` INT primary key,
+    name STRING,
+    description STRING
+) WITH (
+    'connector' = 'jdbc-inlong',
+    'url' = 'jdbc:clickhouse://clickhouse:8123/default',
+    'table-name' = 'test_output1',
+    'username' = 'default',
+    'password' = ''
```

Comment:
Add option "dialect-impl"="org.apache.inlong.sort.jdbc.dialect.ClickHouseDialect". It will can find custom dialect

Reply:
I understand what you mean, but isn't it better to be able to identify the dialect based on the url? It's not an elegant way to test the need to add dialect configuration.

Reply:
> I understand what you mean, but isn't it better to be able to identify the dialect based on the url? It's not an elegant way to test the need to add dialect configuration.

clickhouse url is ok. posrgres variant(greenplum, gaussdb, tdsqlpostgresql) url will same. So I just suggest keep unify.

Reply:
ok, it's been resolved